src/app/pages/BorrowPage/components/AddCollateralModal/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { bignumber } from 'mathjs';
+
+export const BORROW_MAINTENANCE_RATIO = '115';
```

Comment:
I am not sure about that. I read the spec (https://sovryn.monday.com/boards/2277102605) and Ororo wrote in comments that users could be already liquidated at 115% and we should use `item.maintenanceMargin` instead (which is hardcoded to 150% currently but I think it's better to be safe than sorry :D ).